README.md

```diff
@@ -164,15 +164,15 @@ Note: docker has not been fully tested, but has been reported to work.
 ### Docker example
 
 ```shell
-envkernel singularity --name=NAME  --pwd --bind /m/jh/coursedata/:/coursedata /path/to/image.simg
+envkernel docker --name=NAME  --pwd --bind /m/jh/coursedata/:/coursedata /path/to/image.simg
 ```
 
 ### Docker mode arguments
 
 General invocation:
 
 ```shell
-envkernel singularity --name=NAME [envkernel options] [singularity options] [image]
+envkernel docker --name=NAME [envkernel options] [singularity options] [image]
```

Comment:
```suggestion
envkernel docker --name=NAME [envkernel options] [docker options] [image]
```

Reply:
I can't commit this suggestion, so I'll merge and do this next.

Reply:
Oh oops I didn't even catch that -- thank you for pointing that out!